projects/1.19/assets/crimsonsteves-more-mobs/crimson_steves_mobs/lang/zh_cn.json

```
@@ -0,0 +1,133 @@
+{
+  "entity.crimson_steves_mobs.animated_temple": "复苏的神庙",
+  "entity.crimson_steves_mobs.big_snow_golem": "大型雪傀儡",
+  "entity.crimson_steves_mobs.chad_creeper_golem": "苦力怕傀儡斗士",
+  "entity.crimson_steves_mobs.chad_villager": "村民壮士",
+  "entity.crimson_steves_mobs.creeckin": "苦力鸡",
+  "entity.crimson_steves_mobs.creeper_golem": "苦力怕傀儡",
+  "entity.crimson_steves_mobs.creepig": "苦力猪",
+  "entity.crimson_steves_mobs.creesteve": "苦力史蒂夫",
+  "entity.crimson_steves_mobs.crude_redstone_golem": "粗糙的红石傀儡",
+  "entity.crimson_steves_mobs.crude_redstone_monstrosity": "粗糙的红石巨兽",
+  "entity.crimson_steves_mobs.cyborg_vindicator": "机械卫道士",
+  "entity.crimson_steves_mobs.derpig": "Derpig",
+  "entity.crimson_steves_mobs.elite_ender_ravager": "精英末影劫掠兽",
+  "entity.crimson_steves_mobs.ender_ravager": "末影劫掠兽",
+  "entity.crimson_steves_mobs.explosive_phantom": "自爆幻翼",
+  "entity.crimson_steves_mobs.flying_chicken": "飞翔的鸡",
+  "entity.crimson_steves_mobs.giant_illager": "巨型灾厄村民",
+  "entity.crimson_steves_mobs.lava_mutant_zombie": "熔岩突变僵尸",
+  "entity.crimson_steves_mobs.minion_redstone_golem": "粗糙的仆从红石傀儡",
+  "entity.crimson_steves_mobs.pavager": "猪掠兽",
+  "entity.crimson_steves_mobs.phantom_tamer": "幻翼驯兽师",
+  "entity.crimson_steves_mobs.pig_brute": "猪蛮种",
+  "entity.crimson_steves_mobs.ravager_enderman": "劫掠末影人",
+  "entity.crimson_steves_mobs.redstone_monstrosity": "红石巨兽",
+  "entity.crimson_steves_mobs.redstone_powered_iron_golem": "红石动力铁傀儡",
+  "entity.crimson_steves_mobs.t_rabus": "疣猪终结者",
+  "entity.crimson_steves_mobs.tamed_phantom": "驯服的幻翼",
+  "entity.crimson_steves_mobs.vehicle_phantom": "坐骑幻翼",
+  "entity.crimson_steves_mobs.zombie_cow": "僵尸牛",
+  "entity.crimson_steves_mobs.zombie_evoker": "僵尸唤魔者",
+  "entity.crimson_steves_mobs.zombie_pig": "僵尸猪",
+  "entity.crimson_steves_mobs.zombie_pillager": "僵尸掠夺者",
+  "entity.crimson_steves_mobs.zombie_ravager": "僵尸劫掠兽",
+  "entity.crimson_steves_mobs.zombie_vindicator": "僵尸卫道士",
+  "entity.crimson_steves_mobs.zombiefied_chad_villager": "僵尸村民壮士",
+  "gamerule.shouldSpawnAdditionalPatrols": "生成更多巡逻队",
+  "gamerule.shouldSpawnAdditionalPatrols.description": "掠夺者巡逻队中出现更多怪物",
```

```suggestion
  "gamerule.shouldSpawnAdditionalPatrols.description": "灾厄巡逻队中出现更多怪物",
```
https://minecraft.fandom.com/zh/wiki/%E7%81%BE%E5%8E%84%E5%B7%A1%E9%80%BB%E9%98%9F
参照minecraftwiki